### PR TITLE
fix: [13794] fix accessability linker2

### PR DESCRIPTION
**Pull request by joel-ciq**
[Kanban link](https://chartiq.kanbanize.com/ctrl_board/48/cards/13794/details/)

**Description of change** Label added to channel object and displayed in UI.
* Changes - Core update to add "label" property to Linker service.

**Description of testing**
Markdown will convert the 1s to the appropriate number.
1. link this branch to https://github.com/ChartIQ/finsemble/tree/13794-fix-accessability-linker
1. Run Finsemble open the 'Welcome Component' 
1. Click on the linker and selected a couple links.
1. You should see the corresponding number to the link you've clicked on appear next to the linker dropdown.

### CODE REVIEW

File: configs/application/config.json [hunk 1]
```diff
@@ -217,4 +217,4 @@
 		"$applicationRoot/configs/application/services.json",
 		"$applicationRoot/configs/application/securityPolicies.json"
 	]
-}
\ No newline at end of file
+}
```
Comment (NinjaManatee) [inline configs/application/config.json]: Probably should drop this change, since it isn't needed and just makes the diff more complicated.

Comment (joel-ciq) [reply configs/application/config.json]: will do.

**Comment by joel-ciq** [comment]
np, will do.

File: assets/css/_windowTitleBar.css [hunk 2]
```diff
@@ -218,6 +218,8 @@ html.desktop-active .fsbl-icon-highlighted {
 
 .linker-group-accessible {
     width: 1em !important;
+    padding-bottom: 1px;
+    margin-bottom: 4px;
 }
```
Comment (WatsonCIQ) [inline assets/css/_windowTitleBar.css]: The white on the light colours are difficult to see while updating for accessibility could we adjust the contrast between the text and background by using the following, what are your views on this @joel-ciq ?
```suggestion
   text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.79);
```

File: assets/css/_windowTitleBar.css [hunk 3]
```diff
@@ -218,6 +218,8 @@ html.desktop-active .fsbl-icon-highlighted {
 
 .linker-group-accessible {
     width: 1em !important;
+    padding-bottom: 1px;
+    margin-bottom: 4px;
```
Comment (WatsonCIQ) [inline assets/css/_windowTitleBar.css]: The white on the light colours are difficult to see while updating for accessibility could we adjust the contrast between the text and background by using the following, what are your views on this @joel-ciq ?
```suggestion
   margin-bottom: 4px;
   text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.79);

**Comment by carterbs** [comment]
@joel-ciq please change the base to planned/3.11.0

**Comment by joel-ciq** [comment]
Np @carterbs . The PR should then be my branch —> develop right?

**Comment by kriswest** [comment]
Rebasing on planned/3.11.0 was NOT correct - this is a PR for a fix into a recipes branch. There will be a further PR for the recipes branch to merge into a planned/3.X.X branch. Trying to unpick the rebase.

**Pull request merged**